src/dialog/dialog.service.ts

```diff
@@ -155,7 +155,9 @@ export class DialogService {
 			}
 			this.dialogRef = null;
 			this.isOpen = false;
-			elementToFocus.focus();
+			if (focusPreviouslyFocusedElement) {
```

Comment:
So if we don't return focus to the triggering element, focus ends up _somewhere_ on the page ... probably not what we want to do

Reply:
So since we don't focus to the dialog, the previous focus is the trigger element. So when you focus on the trigger the focus gets stuck there.

Reply:
But we need to focus the dialog, so we can access potential interactive elements

Reply:
I mean you can still interact with the elements inside without having the dialog focusable

Reply:
Not with a keyboard AFAIK?

Reply:
If it's not focused we can't really tab into it right?

Reply:
Like for the tooltip with the template, if you are focused on the trigger element and press enter, you can tab to focus on the close button.. so it seems to be working

Reply:
Hmm, it looks like it is, but when I tested it closer it was just cause there were no other tabbable elements on the page ... if I add a button (for example) after the tooltip span, it tabs there _then_ into the tooltip